src/main/kotlin/io/knotx/maven-publish.gradle.kts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2019 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.knotx
+
+plugins {
```

Comment:
Please disable signing for building to maven local.

Reply:
I'm kinda worried about the `val subProjectPath = this.path` in current conditional signing implementation - `this` may point to the wrong context, I'll push it anyway and verify later.